openwisp_network_topology/tests/test_admin.py

```diff
@@ -355,7 +355,7 @@ def test_topology_organization_fk_autocomplete_view(self):
         data = self._create_multitenancy_test_env()
         self._test_multitenant_admin(
             url=self._get_autocomplete_view_path(
-                self.app_label, self.app_label, 'organization'
```

Comment:
@nemesisdesign @Aryamanz29 the `self.app_label` will be `sample_topology` for the sample app. But, there would be no `sample_topology` field/model in the sample app. Therefore, the tests were failing for the sample app.

Reply:
Thank you!

Reply:
Thank you @pandafy!